src/core/containers/OperationContainer.jsx

```diff
@@ -113,11 +113,11 @@ export default class OperationContainer extends PureComponent {
     layoutActions.show(["operations", tag, operationId], !isShown)
   }
 
-  onTryoutClick =() => {
+  onCancelClick=() => {
     this.setState({tryItOutEnabled: !this.state.tryItOutEnabled})
   }
 
-  onCancelClick =() => {
+  onTryoutClick =() => {
```

Comment:
Why the swap here?

Reply:
https://github.com/swagger-api/swagger-ui/commit/3f06bb50fb31951f83dd6b78fa20aee77b6169f7